FC-0068 docs: How-to cleaning by edunext #9 (#714)
openedx/docs.openedx.org: source/educators/how-tos/course_development/add_course_live.rst

Provider specific configuration
********************************

> At present, Open edX supports following video conferencing LTI provider:

Reviewer: ```suggestion
At present, the Open edX platform supports following video conferencing LTI provider:
```

   a. Add "instructor_email" to `Email or Employee Unique ID Attribute Name` setting.

>    b. Add the link to your Open edX instance to the `Approved Domains` setting.

Reviewer: ```suggestion
   b. Add the link to your Open edX instance's Learning MFE (for example, `learning.myinstance.com` - see the URL when you view your course) to the `Approved Domains` setting.
```

   :alt: Screenshot of Zoom rendered in Live tab in course navigation

> Big Blue Button Setup

Reviewer: ```suggestion
Big Blue Button setup
```
